### ISSUE TITLE
Video Player delay

### ISSUE THREAD

**Issue by pitoi** [stakwork/sphinx-ios#132]
Stub: There is up to a 20-30 second delay before the video plays. I don't think it's related to network. It'd be good to find out where the delay is coming from. Other embedded players seem to work fine in the browser.

**Comment by tomastiminskas** [comment]
@pitoi is it on a YouTube video?

**Comment by pitoi** [comment]
yes, i haven' tested with other videos, I'm not sure if we can load other videos at this point (vimeo?)

**Comment by tomastiminskas** [comment]
I have a test tribe with a bitcoin tv RSS feed in it. And at that moment I built the generic video player. But all depends on tribes server being able to handle the RSS feed source and returning the right data structure to the app when it fetches the feed content from the url (that's a separated topic)

Regarding the YouTube videos, I was testing and for me they take the normal time to start playing (like 2/3 seconds at most). I can send a screen recording. I will research a bit when I'm at a good stopping point but as we are using the YouTube library, I don't think we have much control over it.
